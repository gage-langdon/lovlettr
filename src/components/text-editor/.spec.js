import React from 'react';
import { shallow } from 'enzyme';

import TextEditor from './index';

describe('Text-Editor Component', () => {
  it('Will render default props', () => {
    const wrap = shallow(<TextEditor onChange={() => {}} />);
    expect(wrap).toMatchSnapshot();
  });

  it('Will render with value passed as prop', () => {
    const wrap = shallow(
      <TextEditor value="<p>testing 123</p>" onChange={() => {}} />
    );
    expect(wrap).toMatchSnapshot();
  });
});
