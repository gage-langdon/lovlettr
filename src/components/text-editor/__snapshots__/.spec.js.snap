// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text-Editor Component Will render default props 1`] = `
<div
  className="text-editor-container d-flex flex-column"
>
  <div
    className="p-3"
  >
    <Quill
      modules={Object {}}
      onChange={[Function]}
      theme="bubble"
      value="<p>Hello <strong>Beautiful</strong>,</p><p><br></p><p>I hope your day is going well!</p><p> </p><p><br></p>"
    />
  </div>
  <small
    className="text-muted mx-auto mt-auto pb-2"
  >
    Select text to view formatting
  </small>
</div>
`;

exports[`Text-Editor Component Will render with value passed as prop 1`] = `
<div
  className=" d-flex flex-column align-items-center"
>
  <div
    className="text-editor-container"
  >
    <Component
      onChange={[Function]}
      theme="bubble"
      value="<p>testing 123</p>"
    />
  </div>
  <sub
    className="mt-2"
  >
    select text to view formatting
  </sub>
</div>
`;
